### PR TITLE
Open in Editor preserving active document rollback

### PR DESCRIPTION
**Pull request by johnny-keker**
This PR changes Open in Editor step so the focus will always be on opened file (as it was before #201).

### CODE REVIEW

**Review by Kirpich30000 — CHANGES_REQUESTED** [review]
It's better just to comment last line `dte.ItemOperations.OpenFile(activeFile.FullName`. Also please provide better name for the PR.

File: VSRAD.Package/Utils/VsEditor.cs [hunk 1]
```diff
@@ -53,21 +53,17 @@ public static void OpenFileInEditor(SVsServiceProvider serviceProvider, string p
...
-                    Assumes.Present(textManager);
+                var lineNumber = GetMarkedLineNumber(path, lineMarker);
 
-                    textManager.GetActiveView2(1, null, (uint)_VIEWFRAMETYPE.vftCodeWindow, out var activeView);
-                    activeView.SetCaretPos(lineNumber, 0);
-                }
+                var textManager = serviceProvider.GetService(typeof(SVsTextManager)) as IVsTextManager2;
+                Assumes.Present(textManager);
 
-                dte.ItemOperations.OpenFile(activeFile.FullName); // preserving old active document
```
Comment (Kirpich30000) [inline VSRAD.Package/Utils/VsEditor.cs:70]: Just comment this line. No other changes needed.

Comment (johnny-keker) [reply VSRAD.Package/Utils/VsEditor.cs:70]: Thanks, implemented in 4c454f4

**Review by Kirpich30000 — APPROVED** [review]
lgtm

**Pull request merged**